Dockerfile

```diff
@@ -1,23 +1,9 @@
-FROM python:3.8.5-slim-buster
-
-ARG TOKEN=none
-ARG WEBHOOK_ENABLED=false
-ARG WEB_URL=none
-ARG GROUPS=none
-
-ENV BOT_DIR    /app
-
-WORKDIR $BOT_DIR
+FROM python:3
+WORKDIR /app
 
 COPY requirements.txt .
-#Install requirements
-RUN pip3 install --upgrade pip &&\
-  pip3 install -r requirements.txt
+RUN pip3 install -r requirements.txt
 
 COPY . .
-#Setup settings.yaml
-RUN mv config/settings.yaml.dist config/settings.yaml &&\
-  python3 settings.py -t ${TOKEN} -w ${WEBHOOK_ENABLED} -u ${WEB_URL} -g ${GROUPS}
```

Comment:
But how all the configuration will be handled?

Reply:
Secrets mustn't be passed as ARGS. Furthermore images mustn't contain configuration files. We will mount